recipe/meta.yaml

```
@@ -16,6 +16,9 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
```

I'm not sure if this is the issue but....
```suggestion
    - cython                                 # [build_platform != target_platform]
    - scikit-build                           # [build_platform != target_platform]
```